docs/src/modules/components/DiamondSponsors.js

```diff
@@ -125,23 +120,18 @@ export default function DiamondSponsors() {
           alignItems: 'center',
           justifyContent: 'center',
           borderRadius: 1,
-          border: '1px dashed',
+          border: '1px solid',
```

Comment:
I dig the older approach with the dashed border. It felt more like 'this is an empty space ready to be taken'. Besides, I think it'd be good to have homepage's button styled similarly, whatever the approach:
<img width="295" alt="Screenshot 2024-03-27 at 15 26 53" src="https://github.com/mui/material-ui/assets/37222944/8170e6a4-9f67-476f-819e-d60fa225487d">